src/utils/string_utils.F90

```diff
@@ -13,10 +14,6 @@ module string_utils
    public :: strlist_get_ind  ! Gets the index of a given string in a list of strings
    public :: date2yyyymmdd    ! convert encoded date integer to "yyyy-mm-dd" format
    public :: sec2hms          ! convert integer seconds past midnight to "hh:mm:ss" format
-   public :: increment_string ! increments a string
-   public :: last_sig_char    ! Position of last significant character in string
```

Comment:
Just double-checking that these routines were removed because they weren't being used anywhere?  (I remember chatting about unused functions but couldn't remember which ones they were).

Reply:
Yup, I couldn't find any reference in the entire checkout of CAM-SIMA including the submodules.  I'll run the tests again to make sure it still builds (though I'm 99% sure I did that the other week when I removed them originally) and I'll also run the regressions when that PR comes in to be 100% sure.